arch/arm/include/asm/arch-am33xx/pru.h

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (C) 2018 Vorne Industries
```

Comment:
Are the copyright dates needed/desired?

Reply:
Since this goes into open source (e.g., the repo is public), I believe it's a good idea to follow the rest of the files in the repo.